app/components/Main.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TextInput,
+  ScrollView,
+  TouchableOpacity,
+} from 'react-native';
+
+import Note from './Note';
+
+export default class Main extends React.Component {
```

Comment:
Is there a reason why you used class-based component?

Reply:
I think endless hours looking at tutorials led me there.

Reply:
I think endless hours watching tutorials led me there.